packages/guide-cue/src/GuideCue.story.tsx

```diff
@@ -40,6 +40,10 @@ const meta: StoryMetaType<any> = {
         'onPrimaryButtonClick',
       ],
     },
+    chromatic: {
+      disableSnapshot: true,
+      delay: transitionDuration.slowest,
```

Comment:
Do we need this on Menu too?

Reply:
And SplitButton?

Reply:
Don't need the delay if we're disabling it

Reply:
So let's RM where it's duplicative.